openapi/components/schemas/Common/CommonKycDocument.yaml

```
@@ -6,14 +6,17 @@ required:
   - status
 properties:
   id:
+    type: string
     readOnly: true
-    allOf:
-      - $ref: ../ResourceId.yaml
+    description: Unique resource ID.
```

nitpick: we should use this format (I'm not if my attempt has the correct meaning here)
```suggestion
    description: ID of the KYC document.
```